src/engagement/store.mjs

```
@@ -165,6 +192,7 @@ export async function appendEngagementAudit(entry) {
 }
 
 export const __test = {
+  compactSentLog,
```

Biome (2.5.6)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)